Emby.Server.Implementations/HttpServer/HttpListenerHost.cs

```diff
@@ -82,16 +84,14 @@ public class HttpListenerHost : IHttpServer, IDisposable
 
         public ServiceController ServiceController { get; private set; }
 
-        public event EventHandler<GenericEventArgs<IWebSocketConnection>> WebSocketConnected;
-
         public object CreateInstance(Type type)
         {
             return _appHost.CreateInstance(type);
         }
 
         private static string NormalizeUrlPath(string path)
         {
-            if (path.StartsWith("/"))
+            if (path.Length > 0 && path[0] == '/')
```

Comment:
wtf, this isn't a full normalization... :) doesn't asp.net have a normalization routine? I personally prefer slow but correct (and secure) code to fast but incorrect 😛 

cc @Artiume @cvium

Reply:
Note that normalization should probably be fixed in a separate PR, so it is not a blocker now.